caitlinSolution.js

```diff
@@ -0,0 +1,88 @@
+var game = {
+  letters: [],
+  guesses: [],
+  lastGuess: "",
+  display: "",
+  wrongGuesses: 0,
+  guessAccuracy: false,
+  rightGuesses: 0,
+  gameInProgress: true,
+  word: "",
+ // choose word -- make this into a prompt soon
+  enterWord: function() {
+    this.word = prompt("What word would you like to play?");
+
+    this.splitWord();
+  },
+
+  // Split word into an array
+  splitWord: function() {
+    this.letters = this.word.split("");
+
+    this.guessLetter();
+  },
+  // Take letter input & store input in guess array
+  guessLetter: function() {
+    this.guessAccuracy = false;
+
+    while (this.gameInProgress === true) {
+      var letter = prompt("What letter would you like to guess?");
+
+      this.guesses.push(letter);
+
+      this.lastGuess = letter;
+
+      this.checkLetter();
+    };
+
+    if (this.rightGuesses === (this.word.length - 1)) {
+      this.endGame();
+    };
+  },
+  // Check guesses array with letters array
+  checkLetter: function() {
+    for (var i=0; i<this.word.length; i++) {
+      if (this.guesses.includes(this.word[i])) {
+        this.display = this.display + this.word[i] + " ";
+
+        if (this.lastGuess === this.word[i]) {
```

Comment:
I don't think this extra variable lastGuess or this conditional is necessary. I think there is a simpler solution to this :)